src/components/search/SearchResults.tsx

```diff
@@ -112,101 +113,124 @@ const SearchResults: FunctionComponent<SearchResultsProps> = ({ serverId = windo
         setBooks([]);
         setPeople([]);
         setCollections([]);
+        setIsLoading(true);
```

Comment:
You could just put `setIsLoading` after the if to reduce the number of calls to it

Reply:
Changed it! Nice catch